modules/Date.js

```diff
@@ -0,0 +1,11 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+function startTime() {
+  const now = DateTime.now();
+  document.getElementById('date').innerHTML = now.toLocaleString(
+    DateTime.DATETIME_MED,
+  );
+  setTimeout(startTime, 1000);
+}
+
```

Comment:
- [ ] [OPTIONAL] You guys have done a good job refactoring your project. It will be nice if you could please refactor this function to an arrow function.

Reply:
Thanks. I'll check on that!